frameworks/CSharp/FastHttpApi/beetlex-fasthttpapi.dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/dotnet:2.1-sdk-stretch AS build
+WORKDIR /app
+COPY Benchmarks .
+RUN dotnet publish -c Release -o out
+
+WORKDIR /app
+COPY --from=build /app/out ./
```

Comment:
This doesn't work for me locally because you're not using multi-stage builds. Can you remove the `COPY` line and change to `WORKDIR /app/out`